multinode_integration_tests/src/mock_blockchain_client_server.rs

```diff
@@ -155,8 +164,16 @@ impl MockBlockchainClientServer {
     }
 
     pub fn start(&mut self) {
-        let listener = TcpListener::bind(SocketAddr::new(localhost(), self.port)).unwrap();
+        let addr = DockerHostSocketAddr::new(self.port_or_local_addr.unwrap_left());
```

Comment:
I think this variable is actually a port number, not 'addr'.

Reply:
Nope, it's a vector of `SocketAddr`s.